<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InfoAparta_Admi</class>
 <widget class="QWidget" name="InfoAparta_Admi">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>663</width>
    <height>444</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>30</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Información Apartamentos</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>380</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
background-color:rgb(26, 255, 10);
color:black;
border:circular
}</string>
   </property>
   <property name="text">
    <string>Añadir</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>380</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
background-color:rgb(255, 11, 3);
color:black;
border:circular
}</string>
   </property>
   <property name="text">
    <string>Salir</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_Precio">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>220</y>
     <width>251</width>
     <height>41</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Precio</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_Details">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>220</y>
     <width>251</width>
     <height>41</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Otros detalles</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_numCuartos">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>251</width>
     <height>41</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Cantidad de cuartos</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_numPisos">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>160</y>
     <width>251</width>
     <height>41</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Cantidad de Pisos</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_ServPub">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>100</y>
     <width>251</width>
     <height>41</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Servicios Públicos</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_servCochera">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>160</y>
     <width>251</width>
     <height>41</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Cochera</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_Label">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>280</y>
     <width>251</width>
     <height>41</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>ID Apartamento</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
